core/v1.0/core-v1.0.md

```diff
@@ -0,0 +1,609 @@
+
+Permissive processors (for example, a language server which wants to provide best-effort attribution even in the face of document errors) MAY choose to process such documents even in the face of conflicts. Such processors SHOULD include the first (in document order) binding in the scope, and reject subsequent bindings. Such processors SHOULD also provide error messages listing *all* {@link}s which are in conflict.
+
+# Name Conventions
+
+Within a core schema, type names and directives which begin with a valid namespace identifier followed by two underscores (`__`) will be [attributed](#sec-Attribution) to the foreign schema bound to that name in the document [scope](#sec-Scope) if one exists.
+
+```graphql example -- using a prefixed name
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")  
+# 1. Schema("link") -> "https://specs.apollo.dev/link/v1.0" (explicit)
+# 2. Directive("link") -> "https://specs.apollo.dev/link/v1.0#@link" (implicit)
+
+#    👇🏽 🌍 https://specs.apollo.dev/link/v1.0/#Purpose
+enum link__Purpose { SECURITY EXECUTION }
+```
+
+If no schema has been linked to that name, it is interpreted as a local name:
+
+```graphql example -- a strange local name
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")  
+# 1. Schema("link") -> "https://specs.apollo.dev/link/v1.0" (explicit)
+# 2. Directive("link") -> "https://specs.apollo.dev/link/v1.0#@link" (implicit)
+
+#    👇🏽 🌍 #myOwn__Purpose
+enum myOwn__Purpose { SECURITY EXECUTION }
+```
+
+Note: GraphQL name conventions strongly suggest against such naming. But amongst the core schema design principles is *universality*—the ability to represent and link *any arbitrary* set of GraphQL schemas, no matter how weird the names in them are.
+
+## Bootstrapping
+
+Documents can {@link} link itself. Indeed, if they MUST do so if they use {@link} at all and are intended to be [fully valid](#sec-Fully-Valid-Core-Schemas):
+
+```graphql example -- bootstrapping {@link}
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")
+```
+
+The {@link} directive for {@link} itself—the "bootstrap"—MUST be the first {@link} in the document. Other directives may precede it, including directives which have been {@link}ed. 
+
+There is otherwise nothing special or restricted about bootstraps. Documents MAY use them to rename {@link}—either with [`as:`](#@link.as) or [`import:`](#@link.import) or both:
+
+```graphql example -- bootstrapping {@link} with a different name
+extend schema
+  @core(url: "https://specs.apollo.dev/link/v1.0", as: "core")
+```
+
+```graphql example -- importing {@link} with a different name
+extend schema
+  @core(url: "https://specs.apollo.dev/link/v1.0", import: [{ name: "@link", as: "@core" }])
+```
+
+```graphql example -- importing the {@link} directive directly
+extend schema
+  @foo(url: "https://specs.apollo.dev/link/v1.0", import: [
+    {name: "@link", as: "@foo"},
+  ])
+```
```

Comment:
Not sure I get the difference between this example and the one just above. Both seem to import `@link` with a different name?